conf/application.conf

```
@@ -203,6 +203,12 @@ deprecated-routes = [
         method = GET
         routeRegex = "^/ni/.{9}/self-employments/X[A-Z0-9]{1}IS[0-9]{11}/end-of-period-statements/obligations\\?from=.{10}&to=.{10}$"
```

```suggestion
        routeRegex = "^/ni/.{9}/self-employments/X[A-Z0-9]{1}IS[0-9]{11}/end-of-period-statements/obligations"
```
If this doesn't match any other routes then this would be best - otherwise the request won't necessarily be matched (e.g. if the query parameters are in the "wrong" order or omitted)

---

This also applies to the route above this (can't suggest changes on that line though - line 199)